packages/lesswrong/server/conversationUnreadMessages.ts

```diff
@@ -0,0 +1,22 @@
+import { createPostgresView } from "./postgresView";
+
+createPostgresView(`
+  CREATE OR REPLACE VIEW "ConversationUnreadMessages" AS
```

Comment:
It's probably possible to make a SQL resolver for `hasUnreadMessages` without this, but having this view makes the join trivial which seems worth it (there's not really any cost apart from architectural complexity).